llvm/test/ThinLTO/X86/call-calloc.ll

```diff
@@ -0,0 +1,39 @@
+; REQUIRES: x86-registered-target
```

Comment:
nit: this is not needed. `X86` directory has a `lit.local.cfg` that covers this requirement.